/ros2_wj_716N_lidar:
  ros__parameters:
    min_ang : -3.14
    max_ang : 3.14
    range_min : 0.0
    range_max : 30.0
    frequency_scan : 1
    frame_id : "laser"